programs/benchmark/Benchmark.cpp

```diff
@@ -598,6 +598,7 @@ int mainEntryClickHouseBenchmark(int argc, char ** argv)
         boost::program_options::options_description desc = createOptionsDescription("Allowed options", getTerminalWidth());
         desc.add_options()
             ("help",                                                            "produce help message")
+            ("verbose",                                                         "print more detailed message")
```

Comment:
`clickhouse-benchmark` is only used for internal testing. We don't really need to "improve" its help output. I guess we don't need to touch Benchmark.cpp at all.

Reply:
done🙂